test-files/cmd/glacial/NonDestructiveConflationTest.sh.stdout

```diff
@@ -1,181 +1,155 @@
-14:52:55.334 INFO  .../hoot/core/cmd/ConflateCmd.cpp( 189) Conflating test-files/Congo_MGCP_Roads_Bridges_subset.osm with test-files/Congo_OSM_Roads_Bridges_subset.osm and writing the output to test-output/NonDestructiveTest/output.osm
```

Comment:
This test uses the same input data as NonContiguousRoadReviewsTest (see comments for that test), hence the output changes.